test/helpers/DataConverterTests.scala

```diff
@@ -0,0 +1,34 @@
+package helpers
+
+import scala.resources.BaseTestingClass
+
+class DataConverterTests extends BaseTestingClass{
+
+  var dataConverter: DataConverter = _
+
+  before {
+    dataConverter = new DataConverter
+  }
+
+  "imageToBinaryData function" should "return the datatype Array[Byte]" in {
+    val filename = "C:/Users/Admin/IdeaProjects/ScalaFineArtApplication/public/images/favicon.png"
+    assert(dataConverter.imageToBinaryData(filename).isInstanceOf[Array[Byte]])
+  }
+
+  "imageToBinary function" should "be able to accept jpeg and png filetypes" in {
+    val png = "C:/Users/Admin/IdeaProjects/ScalaFineArtApplication/public/images/favicon.png"
+    val jpeg = "C:\\Users\\Admin\\IdeaProjects\\ScalaFineArtApplication\\test\\scala\\resources\\testImages\\test.jpg"
```

Comment:
the string value should be a constant, also try to get it to work with a relative path otherwise once it goes onto jenkins it will break